config/fcos/v1_5_exp/validate.go

```diff
@@ -51,3 +51,14 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (disk Disk) Validate(c path.ContextPath) (r report.Report) {
```

Comment:
Take a look at the `Config` object and see https://github.com/coreos/butane/pull/436#discussion_r1131501088